docs/source/conf.py

```diff
@@ -632,6 +632,78 @@ def create_return_admonitions() -> dict[str, str]:
 RETURN_ADMONITION_FOR_CLASS_NAME = create_return_admonitions()
 
 
+def create_use_in_admonitions() -> dict[str, str]:
+    """Creates 'Use in' admonitions for classes whose instances that are accepted as arguments for
+    Bot's methods.
+    """
+
+    # First, generate a mapping of class names to the methods which accept them as arguments,
+    # i.e. {InlineQueryResult: [answer_inline_query, ...]}
+    methods_for_class_name = defaultdict(set)  # using set here because there can be repetitions
+
+    forward_ref_pattern = re.compile(r"ForwardRef\('(?P<class_name>.+?)'\)")  # non-greedy
+    skip_pattern = re.compile(
+        r"^typing.Union\[ForwardRef\('DefaultValue\[DVType]'\), ((float)|(bool)|(int)|(str))"
+        r"(, NoneType)?]$"
+    )
```

Comment:
more verbose explanations of the patterns would again be nice - I fear that in two weeks nobody will know what these do 😄

Reply:
The second one is irrelevant (I removed it). I'll add a small note on the first one